index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+    <nav class="navBar">
+        <img src="imgs/My Logo.svg" alt="logo" class="logo">
+        <img src="imgs/Union.svg" alt="menu burger" class="menuIcon">
+        <ul class="navInfo">
+            <li>Portoflio</li>
+            <li>About</li>
+            <li>Contact</li>
+            <li><img src="imgs/message.svg" alt="message incon"></li>
+        </ul>
+    </nav>
```

Comment:
- [ ] I would suggest you please wrap this section with the `header` tag typically the header tag is used as a page header.